### ISSUE TITLE
Request for problem_type of training data

### ISSUE THREAD

**Issue by kim1ng** [NVlabs/Long-RL#23]
@yukang2017

Great Work!!! Thanks for releasing the dataset.

Could you please provide the `problem_type` labels for the training set? I noticed that in the dataset, only `test.jsonl` has detailed `problem_type` annotations, while the rest are all marked as "general".
